libvmaf/src/output.c

```diff
@@ -303,7 +303,7 @@ int vmaf_write_output_sub(VmafFeatureCollector *fc, FILE *outfile,
                 continue;
             if (!fc->feature_vector[j]->score[i].written)
                 continue;
-            fprintf(outfile, "%s: %.6f|",
+            fprintf(outfile, "%s: %.30f|",
```

Comment:
we can't unilaterally change this as it breaks existing VMAF tests, I think we'll need to rescale the ST-RRED scores to something more reasonable in the feature extractor instead. But we can follow up on that separately.